packages/theme/src/puik-alert.scss

```diff
@@ -54,4 +54,7 @@
   &__close {
     @apply ml-4 cursor-pointer leading-none;
   }
+  &__link {
+    @apply m-2 max-w-xs whitespace-nowrap overflow-hidden;
```

Comment:
```suggestion
    @apply ml-9 md:m-0 pl-0 md:pl-2 p-2 max-w-xs whitespace-nowrap overflow-hidden;
```

Reply:
hello @kevin-carangeot , the underline of the link does not appear on hover because of the margin.
I also added a left margin for the mobile version (to align it with the button)